client/src/App.tsx

```diff
@@ -15,7 +15,7 @@ function App(): JSX.Element {
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
         <SnackBarProvider>
-          <AuthProvider>
+          {/*<AuthProvider>*/}
```

Comment:
any reason to comment AuthProvider?

Reply:
So I can access http://localhost:3000/dashboard without being rerouted to the login page.

Reply:
^Same as my comment below